python/xorbits/_mars/storage/filesystem.py

```diff
@@ -207,44 +207,56 @@ class JuiceFSStorage(FileSystemStorage):
 
     def __init__(
         self,
-        root_dir: str,
-        local_environ: bool,
+        root_dirs: List[str],
+        is_k8s: bool = False,
+        local_environ: bool = False,
```

Comment:
what  does `local_environ` mean? It would be better to documenting these paramters.